python/ray/tests/test_client_warnings.py

```diff
@@ -18,20 +18,19 @@ class LoggerSuite(unittest.TestCase):
     """Test client warnings are raised when many tasks are scheduled"""
 
     def testOutboundMessageSizeWarning(self):
-        with ray_start_client_server() as ray:
+        with self.assertWarns(UserWarning) as cm, ray_start_client_server() as ray:
```

Comment:
pytest hits a bug when `assertWarns` context is nested; I rewrote to avoid that bug while keeping the same test semantic

Reply:
is it possible to split this small part into a separate PR?